projects/VRDR/DeathRecord_submissionProperties.cs

```diff
@@ -3837,6 +3837,326 @@ public string BirthRecordYear
             }
         }
 
+             
+
+        /// <summary>Birth Record Identifier Child.</summary>
+        /// <value>a birth record child identification string.</value>
```

Comment:
Add comment to describe what this field links, this one is for linking a maternal death to the birth record correct?